custom-tests.yaml

```diff
@@ -879,6 +879,17 @@ api:
   HTMLFontElement:
     __base: var instance = document.createElement('font');
     __test: return bcd.testObjectName(instance, 'HTMLFontElement');
+  HTMLFormControlsCollection:
+    __base: >-
+      <%api.HTMLFormElement:form%>
+      for (var i = 0; i < 3; i++) {
+        var input = document.createElement('input');
+        input.type = "radio";
```

Comment:
The properties aren't necessary to set for this test, right?

It's used for the next test, but better then to not reuse the code I think, so that each test is close to minimal for what is being tested.